static-analysis/teammates-macker.xml

```diff
@@ -287,12 +287,6 @@
                             <from class="${testCases}.browsertests.**" />
                             <to class="${e2e}.**" />
                         </allow>
-
-                        <!-- need to figure out the relation between E2E and client -->
```

Comment:
@wkurniawan07 It turns out that we only need the `targetUrl` in the client package.

Reply:
Good job for finally separating them out, I have been wondering for the longest time whether we need to do it.